core/src/cli_commands/bootstrap.rs

```diff
@@ -451,7 +424,7 @@ pub fn perform_bootstrap_chained_transaction(
 
     let target_balance = calc_target_balance(num_transactions_left)?;
     let box_selector = SimpleBoxSelector::new();
-    let mut inputs = filter_tx_outputs(signed_mint_reward_tokens_tx.outputs.clone());
+    let mut inputs = filter_tx_outputs(signed_pool_box_tx.outputs.clone());
```

Comment:
Thanks for the bug fix!

Reply:
My pleasure. I was happy to see that simulation was able to catch this.